### DISCUSSION TITLE
动态主题theme中的style文件丢失

### DISCUSSION THREAD

**Discussion by Ading163** [ant-design/ant-design#39358] [Q&A] [answered]
**有朋友遇到同样的问题吗？antd版本为最新 5.0.3**
![image](https://user-images.githubusercontent.com/94432082/206136195-d4eea793-4561-4ebc-9596-fc4ab794d320.png)
![image](https://user-images.githubusercontent.com/94432082/206136278-70168e52-9f93-4fbc-8083-794f6a5e243a.png)
![image](https://user-images.githubusercontent.com/94432082/206136352-f17656c8-f2d0-4150-a4b0-caa6574a6157.png)

**Comment by zombieJ** [comment, marked as answer]
https://ant-design.antgroup.com/docs/react/migration-v5-cn#%E7%A7%BB%E9%99%A4-babel-plugin-import

**Comment by yoyo837** [comment]
https://github.com/ant-design/ant-design/issues/39344
